package.json

```diff
@@ -14,7 +14,7 @@
   "author": "Nimbella Corporation",
   "license": "Apache-2.0",
   "dependencies": {
-    "@adobe/aio-cli-plugin-runtime": "nimbella/aio-cli-plugin-runtime#v2021-04-21-1",
+    "@adobe/aio-cli-plugin-runtime": "nimbella/aio-cli-plugin-runtime#v2021-04-21-2",
```

Comment:
We’re using a branch and not a tag??

Reply:
the syntax works for branches and tags equally. I had checked @ https://github.com/nimbella/aio-cli-plugin-runtime/branches/all, it had no branch for `v2021-04-21-1`, but tag and release with same name was there. I created a tag and release for `v2021-04-21-2`.

Reply:
Ah. Thx. We need a break updated tag after this the dev branch is a lot
more recent.